src/inc/payment-gateways/load.php

```diff
@@ -3,26 +3,22 @@
 add_action(
 	'woocommerce_init',
```

Comment:
I wonder if it would also be a good idea to use a "lower priority" (run later) after other woocommerce hooks have fired...

Reply:
Which priority would you recommend?

Reply:
I've changed it to priority 1000 for now.